examples/va/nfv/ovs-dpdk/edpm/values.yaml

```diff
@@ -43,10 +44,15 @@ data:
         # These vars are edpm_network_config role vars
         edpm_network_config_hide_sensitive_logs: false
         edpm_network_config_os_net_config_mappings:
-          edpm-compute:
-            nic2: eno2
-            nic3: eno3
-            nic4: eno4
+          # Need to provide nic mapping based on system uuid or system product name
+          # here used systetm uuid
+          edpm-compute-0: # CHANGEME
+            dmiString: "system-uuid" # CHANGEME
```

Comment:
Are `dmiString` and `id` mandatory in this case? If not, should the user remove them before applying?
@rebtoor fyi

Reply:
@rebtoor yes madatory this case if we want to use interface name. otherwise need to use MAC address and NIC number mapping.

Reply:
How do you acquire the `id` for use with this?

Reply:
Used command for uuid and product name,
# dmidecode --string system-uuid
`# dmidecode --string system-product-name`

Reply:
Ack, thank you @Jaganathancse

Reply:
@Jaganathancse care to send out a doc patch to add those commands to `os-net-config-mapping` module documentation [1] ?

[1] https://openstack-k8s-operators.github.io/edpm-ansible/collections/osp/edpm/edpm_os_net_config_mappings_module.html#ansible-collections-osp-edpm-edpm-os-net-config-mappings-module